cmd/erasure-metadata-utils.go

```diff
@@ -140,7 +140,14 @@ func readVersionFromDisks(ctx context.Context, disks []StorageAPI, bucket, objec
 			}
 			metadataArray[index], err = disks[index].ReadVersion(ctx, bucket, object, versionID, checkDataDir)
 			if err != nil {
-				if err != errFileNotFound && err != errVolumeNotFound && err != errFileVersionNotFound {
+				if !IsErr(err, []error{
+				    errFileNotFound,
+				    errVolumeNotFound,
+				    errFileVersionNotFound,
+				    errDiskNotFound,
+				}{
+				           logger.LogOnceIf(ctx, err, disks[index].String())
+				           }
 					logger.GetReqInfo(ctx).AppendTags("disk", disks[index].String())
```

Comment:
That was a pseudo code you would have to fix the build perhaps..

Reply:
:-D I am surprised/shocked @harshavardhana